go-bindings/Makefile

```
@@ -1,29 +1,50 @@
 -I$(CURR_DIR)../depends/relic/include \
--I$(CURR_DIR)../include"
+-I$(CURR_DIR)../include \
+-I$(GMP_PREFIX)/include"
+
+CGO_LDFLAGS ?= "\
+-L$(CURR_DIR)../build/depends/mimalloc -lmimalloc-secure \
+-L$(CURR_DIR)../build/depends/relic/lib -lrelic_s \
+-L$(CURR_DIR)../build/src -ldashbls \
+-L$(GMP_PREFIX)/lib -lgmp"
+
+.PHONY: default vet test clean config
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)